src/utils/front/scroll.ts

```diff
@@ -0,0 +1,61 @@
+import scroll from 'scroll'
```

Comment:
it feels that we just put everything into utils that are not components, not sure this is right

Reply:
What do you suggest? Where we should store such util functions in another case?